salt/metalk8s/kubernetes/apiserver/installed.sls

```diff
@@ -1,15 +1,16 @@
-{%- from "metalk8s/registry/macro.sls" import kubernetes_image, build_image_name with context -%}
-{% from "metalk8s/map.jinja" import networks with context %}
-{% from "metalk8s/map.jinja" import defaults with context %}
-{% set htpasswd_path = "/etc/kubernetes/htpasswd" %}
+{%- from "metalk8s/registry/macro.sls" import kubernetes_image, build_image_name with context %}
+{%- from "metalk8s/map.jinja" import networks with context %}
+{%- from "metalk8s/map.jinja" import defaults with context %}
+
+{%- set htpasswd_path = "/etc/kubernetes/htpasswd" %}
 
 {% set keepalived_image = "keepalived" %}
```

Comment:
Since you seem to like it, why not adding dashes here too ? :)

Reply:
Was not here when I did the refactoring just appear after a rebase but I will :)